nautobot_firewall_models/forms.py

```diff
@@ -577,6 +577,222 @@ class Meta:
         ]
 
 
+# NATPolicy
+
+
+class NATPolicyRuleFilterForm(BootstrapMixin, StatusModelFilterFormMixin, CustomFieldModelFilterFormMixin):
```

Comment:
There are a few more fields on the model that I think would make sense to enable filtering on. For example, I think the mode and logging are both settings that would be useful to filter on.

Reply:
- Mode is removed
- Logging is following the established pattern from the `PolicyRule` model